core/blockchain.go

```diff
@@ -1465,12 +1467,18 @@ func (bc *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 	}
 	// Ensure the user sees large reorgs
 	if len(oldChain) > 0 && len(newChain) > 0 {
-		logFn := log.Debug
+		logFn := log.Info
+		msg := "Chain reorg detected"
 		if len(oldChain) > 63 {
+			msg = "Chain split detected"
```

Comment:
Might be better (less confusing) to re-use the same terminoloty. So `Chain reorg detected` and `Large chain reorg detected`. Otherwise people will be confused over `split` vs `reorg`

Reply:
thank you for your suggestion! fixed and rebased